src/App.tsx

```diff
@@ -15,71 +18,75 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type AppState = {
+  isSelected: string
```

Comment:
Name this property **selectedGood**, **isSelected** would be suitable if the type of this property is a boolean